_posts/en/newsletters/2020-05-20-newsletter.md

```diff
@@ -0,0 +1,181 @@
+  contacting the BIP341 authors directly.
+
+## News
+
+- **Two-transaction cross chain atomic swap or same-chain coinswap:**
+  Ruben Somsen [posted][somsen post] to the Bitcoin-Dev mailing list and
+  [created a video][somsen video] describing a procedure for a trustless
+  exchange of coins using only two transactions, named Succinct Atomic
+  Swaps (SAS).  The [previous standard protocol][nolan swap] uses four
+  transactions.  Somsen's SAS protocol leaves each party holding coins
+  that they can spend at any time---but which they may need to spend on
+  short notice if their counterparty attempts theft (similar to how LN
+  channels need to be monitored).  Additionally the keys necessary to
+  spend the coins won't be contained in the user's [BIP32][] HD wallet,
+  so additional backups may be required.
+
+    Advantages of the protocol are that it requires less block space
+    than existing protocols, it saves on transaction fees (both by using
+    less block space and potentially by requiring less urgency for its
+    settlement transactions), it only requires consensus-enforced
+    timelocks on one of the chains in a cross-chain swap, and that it
+    doesn't depend on any new security assumptions or Bitcoin consensus
+    changes.  If taproot is adopted, it can be used even more privately
+    and efficiently.
+
+    Commenting on the protocol, Lloyd Fournier [noted][fournier
+    elegance] the "elegance" of a simplified version of the protocol
+    that uses three transactions.  Dmitry Petukhov [posted][petukhov
+    tla+] about a [specification][sas tla+ spec] he'd written for the
+    protocol in the [TLA<sup>+</sup> formal specification language][tla+
+    lang], helping to test the correctness of the protocol.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Lightning-based messenger application Juggernaut launches:**
+  In a [blog post][juggernaut blog] announcing the first release of Juggernaut,
+  John Cantrell describes how the messaging and wallet features are built using
+  [keysend payments][topic spontaneous payments].
+
+- **Lightning Loop using multipath payments:**
+  The latest [upgrade][lightning loop mpp blog] from Lightning Labs
+  now uses [multipath payments][topic multipath payments] to convert onchain
+  funds into funds within LN channels.
+
+- **Blockstream Satellite 2.0 supports initial block download:**
+  Blockstream [outlines version 2.0 upgrades][blockstream satellite v2 blog] to
+  their satellite service which include expanded Asia-Pacific coverage,
+  additional bandwidth, and an updated protocol that enabling a full node to complete an initial sync
+  using only the satellite feed.
+
+- **Breez wallet enables spontaneous payments:**
+  [Version 0.9][breez 0.9] of Breez wallet adds the ability to send spontaneous
+  payments to Lightning nodes that support keysend.
```

Comment:
maybe?: [keysend payments][topic spontaneous payments].

Reply:
Only wanted to avoid because we used it a couple bullets earlier.